Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.py

```diff
@@ -312,102 +405,42 @@ def resp_to_hr(response: dict, type_: str, expanded: bool = False) -> dict:
-            table_for_md = resp_to_hr(response=antivirus, type_="antivirus")
-            readable_output = tableToMarkdown(
-                name=f"Antivirus Reputation: {antivirus.get('id')}",
-                t=table_for_md,
-                removeNull=True,
-            )
-            command_results_list.append(
-                CommandResults(
-                    outputs_prefix="ThreatVault.Antivirus",
-                    outputs_key_field="id",
-                    outputs=antivirus
-                    if expanded
-                    else response.get("data", {}).get("antivirus", []),
-                    readable_output=readable_output,
-                )
-            )
-
-    if "spyware" in response["data"]:
-        if expanded:
-            spywares = response.get("data", {}).get("spyware", [])
-        else:
-            spywares = [response.get("data", {}).get("spyware", ([],))[0]]
-
-        for spyware in spywares:
-            table_for_md = resp_to_hr(response=spyware, type_="spyware")
-            readable_output = tableToMarkdown(
-                name=f"Spyware Reputation: {spyware.get('id')}",
-                t=table_for_md,
-                removeNull=True,
-            )
-            command_results_list.append(
-                CommandResults(
-                    outputs_prefix="ThreatVault.Spyware",
-                    outputs_key_field="id",
-                    outputs=spyware
-                    if expanded
-                    else response.get("data", {}).get("spyware", []),
-                    readable_output=readable_output,
-                )
-            )
-
-    if "vulnerability" in response["data"]:
-        if expanded:
-            vulnerabilities = response.get("data", {}).get("vulnerability", [])
-        else:
-            vulnerabilities = [response.get("data", {}).get("vulnerability", ([],))[0]]
+    for rep_type, headers_type in reputation_types:
+        if rep_type in response["data"]:
+            if expanded:
+                resp = response.get("data", {}).get(rep_type, [])
+            else:
+                resp = [response.get("data", {}).get(rep_type, ([],))[0]]
 
-        for vulnerability in vulnerabilities:
-            table_for_md = resp_to_hr(response=vulnerability, type_="vulnerability")
-            readable_output = tableToMarkdown(
-                name=f"Vulnerability Reputation: {vulnerability.get('id')}",
-                t=table_for_md,
-                removeNull=True,
-            )
-            command_results_list.append(
-                CommandResults(
-                    outputs_prefix="ThreatVault.Vulnerability",
-                    outputs_key_field="id",
-                    outputs=vulnerability
-                    if expanded
-                    else response.get("data", {}).get("vulnerability", []),
-                    readable_output=readable_output,
+            rep_type_readable = convert_reputation_type_to_readable(rep_type)
```

Comment:
we have `resp_to_hr`, let's call this one `reputation_type_to_hr` for convention